db/structs.go

```diff
@@ -910,6 +910,25 @@ type FilterStattuCount struct {
 	Failed    int64 `json:"failed"`
 }
 
+type WfRequestStatus string
+
+const (
+	StatusNew       WfRequestStatus = "NEW"
```

Comment:
@aliraza556 I don't think `NEW` is a status.

Reply:
@elraphty, for now `NEW` is a valid state this will be the default state before we send to stakwork. When stakwork sends back a project_id then we can update it to `PENDING`. I and @humansinstitute agreed to this, but if you have a better status name that will be nice we just came up with that on the fly.

Reply:
Oh! Okay bro, will merge now.